docs/walk-through/README.md

```diff
@@ -0,0 +1,114 @@
+# "Step-by-step Walk-Through" or 
+# How to Use a Hardware Wallet with Bitcoin Core Wallet
+
+Summary: On this page we describe step-by-step and show in screenshots how to use a hardware wallet and HWI 
+together with a Bitcoin Core Wallet. As hardware wallet example we have used a Trezor.
+
+## Create a watch-only Bitcoin Core wallet for Trezor
+Create your watch-only Bitcoin Core Wallet as well described in [Using Bitcoin Core with Hardware Wallets](../bitcoin-core-usage.md). 
+You find all the details in this link. But in summary, one opens one terminal and runs `bitdoind`. E.g.
+
+```
+bitcoind -testnet  -datadir=$HOME/.bitcoin-testnet -rpcallowip=127.0.0.1 -rpcuser=test -rpcpassword=somedifficultpassword
+```
+for a testnet `bitcoind` daemon, or
+```
+bitcoind -rpcallowip=127.0.0.1 -rpcuser=test -rpcpassword=somedifficultpassword
+```
+for a mainnet, i.e. regular, `bitcoind` daemon.
+
+Then in another terminal run commands similar to these, adapted to your environment:
+```
+hwi.py enumerate # this shows you the fingerprint of hardware wallet
+TREZORFINGERPRINT_TESTNET="yourTrezorFingerprint" # shown by "hwi enumerate"
+# in this example we use SEGWIT BECH32 ADDRESSES
+DERIVATIONPATH_TESTNET=1 # testnet uses derivation paths like m/84h/1h/0h/0/* and m/84h/1h/0h/1/* on testnet
+DERIVATIONPATH_MAINNET=0 # mainnet uses derivation paths like m/84h/0h/0h/0/* and m/84h/0h/0h/1/* on testnet
+# if the mainnet path is used on testnet, it will work too but Trezor device gives warnings 
+# of unknown address on Trezor display
+wallet=wallet.test
+rec=$(hwi --testnet -f $TREZORFINGERPRINT_TESTNET getkeypool --wpkh --path m/84h/${DERIVATIONPATH_TESTNET}h/0h/0/* --keypool 0 1000)
+chg=$(hwi --testnet -f $TREZORFINGERPRINT_TESTNET getkeypool --wpkh --path m/84h/${DERIVATIONPATH_TESTNET}h/0h/1/* --keypool --internal 0 1000)
+while ! bitcoin-cli -testnet -rpcuser=test -rpcpassword=somedifficultpassword -rpcclienttimeout=0 createwallet "$wallet" true ; do sleep 60; done
+while ! bitcoin-cli -testnet -rpcuser=test -rpcpassword=somedifficultpassword -rpcclienttimeout=0 -rpcwallet="$wallet" importmulti "$rec" ; do sleep 60; done
```

Comment:
Having these commands be in a `while` loop is unnecessary and could have unintended side effects (re-imports may have some issues).

Reply:
Again, this was taken from a real world script. When doing tests with failing networks, inkecting other faults, etc. the `while` loop is useful. It checks the return code anyway to avoid duplicating the imports. 

Anyway this might not be needed for the average user. I followed your advice and removed the while loops as suggested.